deploy/operator/crd/healthcheck-crd.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: healthchecks.aspnetcore.ui
+
+spec:
+  group: aspnetcore.ui
+  names:
+    plural: healthchecks
+    singular: healthcheck
+    kind: HealthCheck
+    listKind: HealthChecks
+    shortNames:
+      - hc
+  versions:
+    - name: v1
+      served: true
+      storage: true
+
+  scope: Namespaced
+
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            name:
+              type: string
+            portType:
```

Comment:
`serviceType` is a better field name - the type is on the service, not on the port. Defaulting to `ClusterIP`, like a Service does, also seems like a better choice than making it required.

Also,

``` yaml
enum:
- ClusterIP
- LoadBalancer
- NodePort
```

is better for defining a CRD field than a pattern.

Reply:
Agree

Reply:
Yeah, it fits better. I agree